packages/website/src/config.tsx

```diff
@@ -113,7 +114,7 @@ export const TECHNOLOGIES = [
   },
   {
     key: 'next',
-    name: 'NextJS',
+    name: 'NextJS 12',
```

Comment:
I know we have starter kits with different versions of next but I don't think we should specify on the website. It's a next tag/category it will just include any versions of the framework